contrib/buildahimage/Containerfile

```diff
@@ -65,6 +65,9 @@ RUN dnf -y makecache && \
 
 ADD ./containers.conf /etc/containers/
 
+# Setup internal Buildah to pass secrets/subscriptions down from host to internal container
+RUN printf '/run/secrets/etc-pki-entitlement:/run/secrets/etc-pki-entitlement\n/run/secrets/rhsm:/run/secrets/rhsm\n' > /etc/containers/mounts.conf
```

Comment:
Not a blocker, but is there any reason for doing it this way instead of creating a `mounts.conf` file in this subdirectory and adding it to the `ADD` on line 66?

Reply:
Nope, but not sure that would be much better.